pkg/pipeline/stream/destination.go

```diff
@@ -104,7 +105,7 @@ func (n *DestinationNode) Run(ctx context.Context) (err error) {
 		n.logger.Trace(msg.Ctx).Msg("writing record to destination connector")
 
 		writeTime := time.Now()
-		err = n.Destination.Write(msg.Ctx, msg.Record)
+		err = n.Destination.Write(msg.Ctx, []opencdc.Record{msg.Record})
```

Comment:
Notice that the plugin protocol now supports batching, but the node is not taking advantage of that yet.